tests/integration/dhcp_test.py

```diff
@@ -219,3 +219,79 @@ def _clean_up():
         os.unlink(DNSMASQ_CONF_PATH)
     except FileNotFoundError:
         pass
+
+
+def test_slave_ipaddr_learned_via_dhcp_added_as_static_to_linux_bridge(dhcp_env
+                                                                       ):
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'dhcpcli',
+                'type': 'ethernet',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': True
+                },
+            }
+        ]
+    }
+
+    netapplier.apply(desired_state)
+
+    current_state = statelib.show_only(('dhcpcli',))
+    client_current_state = current_state[INTERFACES][0]
+    dhcpcli_ip = client_current_state['ipv4']['address']
+
+    bridge_desired_state = {
+        INTERFACES: [
+            {
+                'name': 'linux-br0',
+                'type': 'linux-bridge',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': False,
+                    'address': dhcpcli_ip
+                },
+                'bridge': {
+                    'options': {
```

Comment:
IMHO it would be nicer to remove all these options to see if it works with the default settings.

Reply:
Done